src/App.scss

```diff
@@ -0,0 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500;700&display=swap');
+
+$theme-color: #0290ff;
+$base-color: #121212;
+$ll-gray: #e8e8e8;
+$gray-border: 1px solid $ll-gray;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Montserrat, sans-serif;
+}
+
+body {
+  color: $base-color;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.roboto {
+  font-family: 'Roboto Slab', serif;
+}
+
+.lb-text {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  line-height: 1.188rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+.dim-text {
+  opacity: 0.5;
+}
+
+.f-center {
+  display: flex;
+  align-items: center;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+header {
+  width: 100%;
+  padding: 0 6.25rem;
+  height: 5.938rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: $theme-color;
+  border-bottom: $gray-border;
+}
+
+.navbar-title {
+  margin-right: 2.938rem;
+  font-size: 1.875rem;
+}
+
+.navigation ul {
+  color: $base-color;
+  letter-spacing: 1.9px;
+}
+
+.navigation ul li {
+  margin-right: 2.563rem;
+}
+
+.active-link {
+  opacity: 1 !important;
```

Comment:
- [ ] Kindle let's avoid using the `!important` since it is a bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets 👍